Source/Orts.Formats.Msts/CabViewFile.cs

```diff
@@ -1149,14 +1157,23 @@ public CVCDiscrete(STFReader stf, string basepath)
                     ControlStyle = CABViewControlStyles.WHILE_PRESSED;
                 if (ControlType == CABViewControlTypes.DIRECTION && Orientation == 0)
                     Direction = 1 - Direction;
+
+                switch (discreteStates)
+                {
+                    case DiscreteStates.TriState:
+                        MaxValue = 2.0f; // So that LocomotiveViewerExtensions.GetWebControlValueList() returns right value to web server
+                        break;
+                    default:
+                        break;
+                }
             }
-//            catch (Exception error)
-//            {
-//                if (error is STFException) // Parsing error, so pass it on
-//                    throw;
-//                else                       // Unexpected error, so provide a hint
-//                    throw new STFException(stf, "Problem with NumPositions/NumValues/NumFrames/ScaleRange");
-//            } // End of Need check the Values collection for validity
+            //            catch (Exception error)
+            //            {
+            //                if (error is STFException) // Parsing error, so pass it on
+            //                    throw;
+            //                else                       // Unexpected error, so provide a hint
+            //                    throw new STFException(stf, "Problem with NumPositions/NumValues/NumFrames/ScaleRange");
+            //            } // End of Need check the Values collection for validity
```

Comment:
Don't keep commented code inside the repository.
Either remove it or put it inside preprocessor directives (#if DEBUG) if it is needed for debug purposes.

Reply:
Agreed, but my intention is to fix a bug, not to review this class. That really requires a new PR.
I've now undone the format change that highlighted this section.